cypress/e2e/tests/migration/applicationinventory/applications/pagination.test.ts

```diff
@@ -33,27 +33,20 @@ describe(["@tier3"], "Application inventory pagination validations", function ()
         applicationsList = createMultipleApplications(11);
     });
 
-    beforeEach("Persist session", function () {
-        // Interceptors for Applications
-        cy.intercept("GET", "/hub/application*").as("getApplications");
-    });
-
     it("Navigation button validations", function () {
         // Navigate to Application inventory tab
-        Application.open(true);
-        cy.wait("@getApplications");
+        Application.open();
         selectItemsPerPage(10);
         validatePagination();
     });
 
     it("Items per page validations", function () {
-        Application.open(true);
-        cy.wait("@getApplications");
+        Application.open();
         itemsPerPageValidation();
     });
 
     after("Perform test data clean up", function () {
-        Application.open(true);
+        Application.open();
```

Comment:
The Application.open(true); in after method is for a reason that if any of the test fails , clean up will still continue. This should not be removed.

Reply:
Done